0a.md

```diff
@@ -0,0 +1,66 @@
+NIP-0a
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of audio track. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+The following kinds are defined by this NIP:
+
+- `31337` represents a music track
+- `31338` represents a podcast episode
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in [NIP 92](./92.md). Multiple `imeta` tags MAY be included to provide for different
+  use cases (e.g. streaming vs download).
+- `title` is the title of the audio track
+- `subject` is the title of the audio track (deprecated, but required for compatibility)
+
+The following tags are optional:
+
+- `i` is an external GUID in the format defined [here](https://github.com/MerryOscar/nips/commit/280eb498e0ac56b8f9356c1b7a88cc8b31579801).
+- `c` is the track's value as defined by a given category.
```

Comment:
This seems too vague. Also, the category should definitely be the mark:

["c", "genre", "Pop"]

not

["c", "Pop", "genre"] <<< this is backwards compared to most other tags :(

Reply:
- genre
- subgenre
- artist
- record_label
- producer

what other fields should be expected?

- composer
- album
- track_number
- tracks
- bpm
- year

![https://images.prismic.io/soundcharts/868e0bf5c60017040f5ca7b84ca94ee1606204d5_macos-itunes-edit-metadata-how-to-2.jpg?auto=compress,format](https://images.prismic.io/soundcharts/868e0bf5c60017040f5ca7b84ca94ee1606204d5_macos-itunes-edit-metadata-how-to-2.jpg?auto=compress,format)

Reply:
https://soundcharts.com/blog/music-metadata might provide helpful insight on doing this right

Reply:
> ["c", "Pop", "genre"] <<< this is backwards compared to most other tags :(

This looks backward, but like with NIP 32 the intention is to be able to index the value, rather than the key. It's more important to be able to filter by category values (e.g. `#c: ["Pop"]`) than to filter by what categories are defined.

Also added a list of categories to the NIP.

Reply:
> It's more important to be able to filter by category values (e.g. #c: ["Pop"]) than to filter by what categories are defined.

That's a good point. Too bad this makes the event.getMatchingTags function in NDK totally useless for parsing these audio events, but I can implement my own tag parsing.

Thanks for adding those categories.